SpatialGDK/Source/SpatialGDKServices/Public/SpatialGDKServicesConstants.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialGDKServicesModule.h"
+
+namespace SpatialGDKServicesConstants
+{
+#if PLATFORM_WINDOWS
+	// Assumes that spatial is installed and in the PATH
+	const FString SpatialPath = TEXT("");
+	const FString Extension = TEXT("exe");
+#elif PLATFORM_MAC
+	// This is currently hardcoded and we expect users to have spatial either installed or symlinked to this path.
+	// If they haven't, it is necessary to symlink it to /usr/local/bin. At some point we should expose this via
```

Comment:
Can we get a ticket in here for this please

Reply:
sure, do you want the ticket also  linked in the comment or just a ticket?

Reply:
Ticket linked in the ticket too please :D